Assets/Scripts/Combat/Projectile.cs

```diff
@@ -25,79 +37,179 @@ public class Projectile : MonoBehaviour
 
     private void Awake()
     {
-      meshRenderer = GetComponent<MeshRenderer>();
+      vfxPooler = FindObjectOfType<VfxPooler>();
+
       PhysicsController = GetComponent<Rigidbody>();
+      if (usesCustomGravity) PhysicsController.useGravity = false;
 
       disableOnEnabled = true;
-
-      explosionVfx.OnDeactivate += Deactivate;
     }
 
     private void OnEnable()
     {
       if (disableOnEnabled) { disableOnEnabled = false; return; }
 
+      disableProjectileCollisions = false;
       BulletEvents.BulletFired(this);
     }
 
+    private void FixedUpdate()
+    {
+      if (usesCustomGravity) PhysicsController.AddForce(0f, customGravityValue, 0f, ForceMode.Acceleration);
```

Comment:
Note to self @noshadowkick2000 this should be adapted for custom gravity in case of planetary idea